### PR TITLE
Change gnome to textmode for sles12sp5 autoyast profile

### PR DESCRIPTION
**Pull request by tinawang123**
Change gnome to textmode for sles12sp5 autoyast profile

- Related ticket: https://progress.opensuse.org/issues/158431
- Verification run: https://openqa.suse.de/tests/14067572
  https://openqa.suse.de/tests/14048315
  https://openqa.suse.de/tests/14048316
- Related MR: https://gitlab.suse.de/qe-yam/openqa-job-groups/-/merge_requests/158

### CODE REVIEW

File: schedule/yast/maintenance/sap_autoyast_create_hdd_textmode_netweaver_sle12.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
+---
+name: sap_autoyast_create_hdd_gnome_netweaver_sle12
```
Comment (jknphy) [inline schedule/yast/maintenance/sap_autoyast_create_hdd_textmode_netweaver_sle12.yaml:2]: textmode

**Review by lemon-suse — APPROVED** [review]

File: schedule/yast/maintenance/sap_autoyast_create_hdd_textmode_netweaver_sle12.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+description: '|
```
Comment (jknphy) [inline schedule/yast/maintenance/sap_autoyast_create_hdd_textmode_netweaver_sle12.yaml:3]: could you edit the existing file for gnome if it is owned by us and we don't use it anywhere else?
if we still need a duplicated file with few adjustment for textmode, could you please put those adjustment out and share the same file (or file another ticket also could work)?

Comment (tinawang123) [reply schedule/yast/maintenance/sap_autoyast_create_hdd_textmode_netweaver_sle12.yaml:3]: We need keep this gnome file, as it needed by x86_64 platform.

File: schedule/yast/maintenance/sap_autoyast_create_hdd_textmode_netweaver_sle12.yaml (hunk 1 above)
Comment (tinawang123) [reply schedule/yast/maintenance/sap_autoyast_create_hdd_textmode_netweaver_sle12.yaml]: Updated

File: schedule/yast/maintenance/sap_autoyast_create_hdd_textmode_netweaver_sle12.yaml (hunk 2 above)
Comment (tinawang123) [reply schedule/yast/maintenance/sap_autoyast_create_hdd_textmode_netweaver_sle12.yaml:3]: I will file another ticket to adjust those two yaml files.

File: data/yam/autoyast/support_images/sles12sp5_install_default_patterns_ppc64le_multi-user.xml [hunk 3]
```diff
@@ -0,0 +1,256 @@
...
+      <product>{{SLE_PRODUCT}}</product>
+    </products>
+    <packages config:type="list">
+    </packages>
+    <patterns config:type="list">
+      <pattern>Minimal</pattern>
+      <pattern>apparmor</pattern>
+      <pattern>base</pattern>
+      <pattern>documentation</pattern>
+      <pattern>gnome-basic</pattern>
```
Comment (jknphy) [inline data/yam/autoyast/support_images/sles12sp5_install_default_patterns_ppc64le_multi-user.xml:235]: why is there any gnome? is this a desktop image that we configure to access by default with multi-user or a real textmode image?

Comment (tinawang123) [reply data/yam/autoyast/support_images/sles12sp5_install_default_patterns_ppc64le_multi-user.xml]: Configured with multi-user

**Pull request merged**